libzfs-sys/build.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2017 Intel Corporation. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+extern crate bindgen;
+extern crate pkg_config;
+
+use std::env;
+use std::path::PathBuf;
+
+fn main() {
+    env::set_var("LIBCLANG_PATH", "/opt/llvm-5.0.0/lib64/");
```

Comment:
Where does `/opt/llvm-5.0.0/lib64/` come from?

Reply:
https://github.com/intel-hpdd/rust-libzfs/blob/d43d297fb76368aea9e59d51f38c10f3e50a4d42/Vagrantfile#L27-L28

The version of clang shipping in EPEL too old for bindgen.